models/model.py

```diff
@@ -0,0 +1,368 @@
+#room base
+class Room():
+    '''
+    models abstract data type Room, not intended to be instanciated
+    '''
+    #keep track of all rooms created
+    number_of_rooms = 0
+
+    def __init__(self, max_occupants, name):
+        '''
+        return Room with no occupats
+        '''
+        Room.number_of_rooms += 1
+        self.__name = name
+        self.__max_occupants = max_occupants
+        self.__occupants = []
+        self.__id = Room.number_of_rooms
+
+
+    @property
```

Comment:
The syntax you have here will return to you the desired results, but python doesn't require getters and setters. I believe there is a better way you could retrieve and set data programatically without the need for all these lines.